### ISSUE TITLE
Not able to access http://<IP>:9601/metrics in GKE cluster using k8 soketi helm.

### ISSUE THREAD

**Issue by nw18rishabhjain** [soketi/charts#36]
I have installed k8 soketi via helm chart and enabled `metrics` and `serviceMonitor` 
However, when I try to access `my-k8-soketi-k8soketi-metrics` service using url `http://<IP>:9601/metrics` it is not accessible via curl.

The metrics service in GKE is not having any endpoint.

<img width="1799" alt="Screenshot 2023-08-18 at 11 30 36 AM" src="https://github.com/soketi/charts/assets/108113158/2d3f93fd-21af-4204-926b-bd3b64aa8a65">

**Comment by rennokki** [comment]
Yes, this is a chart bug that will be fixed. To workaround, it requires you to set the `SOKETI_METRICS_ENABLED=1` environment variable with `extraEnv`
